api/Element.json

```diff
@@ -6269,6 +6269,234 @@
           }
         }
       },
+      "transitioncancel_event": {
+        "__compat": {
+          "description": "<code>transitioncancel</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/transitioncancel_event",
+          "spec_url": [
+            "https://drafts.csswg.org/css-transitions/#transitioncancel",
+            "https://drafts.csswg.org/css-transitions/#ref-for-dom-globaleventhandlers-ontransitioncancel"
+          ],
+          "support": {
+            "chrome": [
+              {
+                "version_added": "87"
+              },
+              {
+                "version_added": "74",
+                "partial_implementation": true,
+                "notes": "The <code>ontransitioncancel</code> attribute is not supported in IE. To listen to this event, use <code>document.addEventListener('transitioncancel', function() {});</code>."
+              }
+            ],
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "53"
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": [
+              {
+                "version_added": "13.1"
+              },
+              {
+                "version_added": "12",
+                "partial_implementation": true,
+                "notes": "The event handler is exposed but will not actually be called unless both the \"Web Animations\" and \"CSS Animations via Web Animations\" preferences are enabled."
```

Comment:
Can you drop the mention of flag data here, and just describe that the event handler exists but the event is never fired?